class/HPGrowingTextView.m

```diff
@@ -114,7 +114,7 @@ -(void)commonInitialiser
 
     [self setPlaceholderColor:[UIColor lightGrayColor]];
     internalTextView.displayPlaceHolder = YES;
-    self.internalTextView.accessibilityId = @"textMessageViewAccessibilityID";
```

Comment:
This isn't the message recipient field, its the message input field - can we make sure the name reflects that

Reply:
self.internalTextView.accessibilityIdentifier = @"Burner_Message_Recipient_Text_Field";